frontend/src/scenes/notebooks/Nodes/utils.tsx

```diff
@@ -60,3 +63,35 @@ export function externalLinkPasteRule(options: {
         },
     })
 }
+
+export function nodePasteRule(config: {
```

Comment:
I had to reimplement the nodePasteRule because it didn't allow for content to be inserted when creating a node. Opened a PR to fix that in a future version: https://github.com/ueberdosis/tiptap/pull/4212